intro.md

```diff
@@ -1,8 +1,40 @@
-# Tskit tutorials
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.9.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
 
-This is the home of tskit tutorials.
+# Tree sequence tutorials
 
+**Welcome!**
 
-:::{warning}
-This site is under heavy development.
+This site contains a number of tutorials to develop your understanding of
+[tree sequences](https://tskit.dev/learn.html#what) and software programs,
+such as [msprime](https://tskit.dev/msprime), that use them.
+
+If you are new to the world of tree sequences, we suggest you start with the
+first tutorial: {ref}`sec_what_is`
+
+We aim to be a friendly, welcoming open source community.
+Questions and discussion about using {program}`tskit`, the tree sequence toolkit
+should be directed to the
+[GitHub discussion forum](https://github.com/tskit-dev/tskit/discussions), and there are
+[similar](https://github.com/tskit-dev/msprime/discussions)
```

Comment:
Not sure this linking structure is helpful, how about ``similar forums for software such as [msprime]() and [tsinfer]()``

Reply:
True, yours is better. Done in latest commit on this PR.